machine/v1beta1/types_awsprovider.go

```diff
@@ -165,6 +165,11 @@ type EBSBlockDeviceSpec struct {
 	// it is not used in requests to create gp2, st1, sc1, or standard volumes.
 	// +optional
 	Iops *int64 `json:"iops,omitempty"`
+	// Throughput to provision in MiB/s. This parameter is valid only for gp3 volumes.
```

Comment:
[CAPI docs][1]:

> Throughput to provision in MiB/s supported for the volume type. Not applicable to all types.

Do we want to echo that punt to AWS?  Or do we want the inlined AWS context you have here?

[1]: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/777e8dee6df33d49a540a7e1cec66df601930e5c/api/v1beta1/types.go#L230-L232

Reply:
I was following this, which I felt was more informative.

https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_EbsBlockDevice.html

 "This only applies to the enumerated volume types" is more helpful than "This doesn't apply to some unenumerated volumes types"

Reply:
it is more helpful while it's accurate, but it means you're on the hook to keep it accurate.  The rate of AWS-side changes is likely not high, so it could go either way.  I'm just pointing out that CAPI has gone the punt-to-AWS way.